components/esp_http_server/include/esp_http_server.h

```diff
@@ -70,6 +70,35 @@ typedef void* httpd_handle_t;
  */
 typedef enum http_method httpd_method_t;
 
+/**
+ * @brief  Prototype for freeing context data (if any)
+ * @param[in] ctx : object to free
+ */
+typedef void (*httpd_free_sess_ctx_fn_t)(void *ctx);
```

Comment:
Since we have agreed to modify the APIs, I guess we can also rename this to `httpd_free_ctx_fn_t` since this is being used for both global and session contexts

Reply:
Sure, but it'll take about 12 hours before I can do this. If you want, please feel free to rename it yourself